### ISSUE TITLE
[Question]: How to display cosine similarity score in vector search results?

### ISSUE THREAD

**Issue by stay-leave** [infiniflow/infinity#2270]
### Describe your problem

I have tried printing the results multiple times using different formats, but none of them displays the cosine similarity score. I would like to know how I can retrieve and display the similarity score for the vector search results.

Here is the relevant part of my code:
```
res = table_text.output(["*"]).match_dense(
        "chunk_embed", np.random.rand(768).tolist(), "float", "cosine", 3,
        {"threshold": "0.3"}
        ).to_arrow()
    print(res)
```
return：
uuid: string
inner_id: int32
title: string
author: string
abstract: string
key_words: string
file_path: string
chunk_content: string
chunk_embed: list<item: double>
  child 0, item: double

**Comment by small-turtle-1** [comment]
you can use `table_obj.output(["_distance"])` to output similarity of cosine distance.
```python
    def test1(self):
        db_obj = self.infinity_obj.get_database("default_db")
        db_obj.drop_table("test1", ConflictType.Ignore)
        table_obj = db_obj.create_table(
            "test1", {"c1": {"type": "int"}, "c2": {"type": "vector,4,float"}}
        )
        test_csv_path = "test/data/csv/embedding_float_dim4.csv"
        table_obj.import_data(test_csv_path)
        res = table_obj.output(["c1", "_distance"]).match_dense(
            "c2", [0.1, 0.1, 0.1, 0.1], "float", "l2", 1
        ).to_df()
        print("\n")
        print(res)
```

**Comment by stay-leave** [comment]
thanks, l2 is ok ! but cosine is not work.

info: SCORE() requires Fusion or MATCH TEXT or MATCH TENSOR
my code:
```
res = table_text.output(["uuid","file_path","chunk_content", "_score"]
                            ).match_dense(
                            "chunk_embed", query_embed, "float", "cosine", k,
                            {"threshold": "0.5"}
                            ).to_df()
```
query_embed is a list like [0.1,0.1]
I try to turn it to torch.tensor, but not run work.

**Comment by JinHai-CN** [comment]
@stay-leave 
- _score: A BM25 score used in full-text search.
- _similarity: Used by IP and cosine metrics in dense or sparse vector search.
- _distance: Used by L2 metric in dense vector search.

From https://infiniflow.org/docs/dev/pysdk_api_reference#search

**Comment by stay-leave** [comment]
谢谢大佬！明白了
